Doc/library/sqlite3.rst

```diff
@@ -1418,15 +1418,22 @@ Cursor objects
 
    .. method:: execute(sql, parameters=(), /)
 
-      Execute SQL statement *sql*.
-      Bind values to the statement using :ref:`placeholders
-      <sqlite3-placeholders>` that map to the :term:`sequence` or :class:`dict`
-      *parameters*.
+      Execute SQL a single SQL statement,
+      optionally binding Python values using
+      :ref:`placeholders <sqlite3-placeholders>`.
 
-      :meth:`execute` will only execute a single SQL statement. If you try to execute
-      more than one statement with it, it will raise a :exc:`ProgrammingError`. Use
-      :meth:`executescript` if you want to execute multiple SQL statements with one
-      call.
+      :param unicode sql:
+         A single SQL statement.
+
+      :param parameters:
+         Python values to bind to placeholders in *sql*.
+         A :class:`!dict` if named placeholders are used.
+         A :term:`!sequence` if unnamed placeholders are used.
+         See :ref:`sqlite3-placeholders`.
+      :type parameters: :class:`dict` | :term:`sequence`
+
+      :raises ProgrammingError:
+         If *sql* contains more than one SQL statement.
```

Comment:
> Also, I'm in doubt of how to use the :raises Sphinx param here; execute*() can raise a heck of a lot of exceptions; should we write all of them down here, or just a subset? Perhaps just point to the reference for sqlite3 exceptions?

Yeah, I feel like documenting exceptions that are frequently raised is Good, but if we leave it like this, we'll inevitably end up with "bug reports" saying "The sqlite documentation promises that only ProgrammingError will be raised, but I got a TypeError!!!1!1" or something along those lines.

Reply:
True that. I'll save that for a follow-up PR, though. It'll take some time.

Reply:
Hmm, I feel like the current wording makes clear that `ProgrammingError` is not the only exception that _could_ be raised, whereas using the `:raises:` Sphinx param could imply that the docs are promising that it's the only exception that will ever be raised. So maybe we should not use `:raises:` here, and leave the language on `ProgrammingError` as it is (at least for now)?

Reply:
Yeah, that's a good idea.

Reply:
OTOH, I can just queue a PR that covers all exceptions later today (or tomorrow), so lets just keep it as it is on the PR.